0006/README.md

```diff
@@ -0,0 +1,49 @@
+---
+id: 0006
+title: Resiliency in Tinkerbell Provisioner.
+status: Ideation
+authors: Chitrabasu Khare <chitrabasukhare89@gmail.com>
+---
+
+## Summary:
+
+This RFD proposes to have a resilient framework for Tinkerbell Provisioner.
+
+
+## Goals:
+
+- Bring resiliency in Tinkerbell. 
+- Tinkerbell as a setup should be highly available.
+
+## Content: 
+
+- We would like to have a framework where Tinkerbell as a solution is fault-tolerant.
+- All most all of the core services are managed by provisioner. So, it will be good to have a framework where provisioner is highly available.
+- It can be a multinode provisioner solution with active-active or active-passive implementation.
+- The Provisioner services should be partition tolerant during network failures or communication failure between provisioner nodes.
```

Comment:
This is a good proposal by its own

Reply:
@detiber I agree with you. 

We can start with DB first and then scale onto other services.

Reply:
I'm getting a similar sense from this proposal, especially with regards to the L3/L7 load balancer recommendation.  I wonder if we could start with "Recommendations for running Tinkerbell in HA", as Tinkerbell is currently provided, and use insights from that exploration to guide proposals for individual enhancements.
 
Generally, I worry that with all of the potential environments Tinkerbell could be run in, any single proposed loadbalancer (DB replication, etc) proposal may be too opinionated. We should do a better job of understanding or segmenting the landscape first and start with a blog post or guide.

For some scenarios (dev, demo, home-lab), HA options are not important.